diff -up Cemu-2.1/src/Cemu/Logging/CemuLogging.h.omv~ Cemu-2.1/src/Cemu/Logging/CemuLogging.h
--- Cemu-2.1/src/Cemu/Logging/CemuLogging.h.omv~	2024-09-02 20:09:11.470337875 +0200
+++ Cemu-2.1/src/Cemu/Logging/CemuLogging.h	2024-09-02 20:24:36.681742357 +0200
@@ -52,7 +52,7 @@ enum class LogType : sint32
 template <>
 struct fmt::formatter<std::u8string_view> : formatter<string_view> {
 	template <typename FormatContext>
-	auto format(std::u8string_view v, FormatContext& ctx) 
+	auto format(std::u8string_view v, FormatContext& ctx) const
 	{
 		string_view s((char*)v.data(), v.size());
 		return formatter<string_view>::format(s, ctx);
@@ -91,7 +91,7 @@ bool cemuLog_log(LogType type, std::basi
 	else
 	{
 		const auto format_view = fmt::basic_string_view<T>(formatStr);
-		const auto text = fmt::vformat(format_view, fmt::make_format_args<fmt::buffer_context<T>>(args...));
+		const auto text = fmt::vformat(format_view, fmt::make_format_args<fmt::buffered_context<T>>(args...));
 		cemuLog_log(type, std::basic_string_view(text.data(), text.size()));
 	}
 	return true;
diff -up Cemu-2.1/src/config/CemuConfig.h.omv~ Cemu-2.1/src/config/CemuConfig.h
--- Cemu-2.1/src/config/CemuConfig.h.omv~	2024-09-02 20:20:56.059113096 +0200
+++ Cemu-2.1/src/config/CemuConfig.h	2024-09-02 20:21:23.949449396 +0200
@@ -194,7 +194,7 @@ ENABLE_ENUM_ITERATORS(CrashDump, CrashDu
 template <>
 struct fmt::formatter<PrecompiledShaderOption> : formatter<string_view> {
 	template <typename FormatContext>
-	auto format(const PrecompiledShaderOption c, FormatContext &ctx) {
+	auto format(const PrecompiledShaderOption c, FormatContext &ctx) const {
 		string_view name;
 		switch (c)
 		{
@@ -209,7 +209,7 @@ struct fmt::formatter<PrecompiledShaderO
 template <>
 struct fmt::formatter<AccurateShaderMulOption> : formatter<string_view> {
 	template <typename FormatContext>
-	auto format(const AccurateShaderMulOption c, FormatContext &ctx) {
+	auto format(const AccurateShaderMulOption c, FormatContext &ctx) const {
 		string_view name;
 		switch (c)
 		{
@@ -223,7 +223,7 @@ struct fmt::formatter<AccurateShaderMulO
 template <>
 struct fmt::formatter<CPUMode> : formatter<string_view> {
 	template <typename FormatContext>
-	auto format(const CPUMode c, FormatContext &ctx) {
+	auto format(const CPUMode c, FormatContext &ctx) const {
 		string_view name;
 		switch (c)
 		{
@@ -240,7 +240,7 @@ struct fmt::formatter<CPUMode> : formatt
 template <>
 struct fmt::formatter<CPUModeLegacy> : formatter<string_view> {
 	template <typename FormatContext>
-	auto format(const CPUModeLegacy c, FormatContext &ctx) {
+	auto format(const CPUModeLegacy c, FormatContext &ctx) const {
 		string_view name;
 		switch (c)
 		{
@@ -257,7 +257,7 @@ struct fmt::formatter<CPUModeLegacy> : f
 template <>
 struct fmt::formatter<CafeConsoleRegion> : formatter<string_view> {
 	template <typename FormatContext>
-	auto format(const CafeConsoleRegion v, FormatContext &ctx) {
+	auto format(const CafeConsoleRegion v, FormatContext &ctx) const {
 		string_view name;
 		switch (v)
 		{
@@ -278,7 +278,7 @@ struct fmt::formatter<CafeConsoleRegion>
 template <>
 struct fmt::formatter<CafeConsoleLanguage> : formatter<string_view> {
 	template <typename FormatContext>
-	auto format(const CafeConsoleLanguage v, FormatContext &ctx) {
+	auto format(const CafeConsoleLanguage v, FormatContext &ctx) const {
 		string_view name;
 		switch (v)
 		{
@@ -304,7 +304,7 @@ struct fmt::formatter<CafeConsoleLanguag
 template <>
 struct fmt::formatter<CrashDump> : formatter<string_view> {
 	template <typename FormatContext>
-	auto format(const CrashDump v, FormatContext &ctx) {
+	auto format(const CrashDump v, FormatContext &ctx) const {
 		string_view name;
 		switch (v)
 		{
@@ -321,7 +321,7 @@ struct fmt::formatter<CrashDump> : forma
 template <>
 struct fmt::formatter<CrashDump> : formatter<string_view> {
 	template <typename FormatContext>
-	auto format(const CrashDump v, FormatContext &ctx) {
+	auto format(const CrashDump v, FormatContext &ctx) const {
 		string_view name;
 		switch (v)
 		{
diff -up Cemu-2.1/src/gui/helpers/wxHelpers.h.omv~ Cemu-2.1/src/gui/helpers/wxHelpers.h
--- Cemu-2.1/src/gui/helpers/wxHelpers.h.omv~	2024-09-02 20:26:43.653229279 +0200
+++ Cemu-2.1/src/gui/helpers/wxHelpers.h	2024-09-02 20:26:48.383284379 +0200
@@ -8,7 +8,7 @@ template <>
 struct fmt::formatter<wxString> : formatter<string_view>
 {
 	template <typename FormatContext>
-	auto format(const wxString& str, FormatContext& ctx)
+	auto format(const wxString& str, FormatContext& ctx) const
 	{
 		return formatter<string_view>::format(str.c_str().AsChar(), ctx);
 	}
diff -up Cemu-2.1/src/input/emulated/EmulatedController.h.omv~ Cemu-2.1/src/input/emulated/EmulatedController.h
--- Cemu-2.1/src/input/emulated/EmulatedController.h.omv~	2024-09-02 20:19:54.048360514 +0200
+++ Cemu-2.1/src/input/emulated/EmulatedController.h	2024-09-02 20:20:28.428778624 +0200
@@ -127,7 +127,7 @@ using EmulatedControllerPtr = std::share
 template <>
 struct fmt::formatter<EmulatedController::Type> : formatter<string_view> {
 	template <typename FormatContext>
-	auto format(EmulatedController::Type v, FormatContext& ctx) {
+	auto format(EmulatedController::Type v, FormatContext& ctx) const {
 		switch (v)
 		{
 		case EmulatedController::Type::VPAD: return formatter<string_view>::format("Wii U Gamepad", ctx);
